docs/management/managing-saved-objects.asciidoc

```diff
@@ -1,81 +1,75 @@
 [[managing-saved-objects]]
-== Managing Saved Objects
+== Saved objects
 
-You can view, edit, delete, import, and export saved objects from 
-*Management > Saved Objects*. A saved object can be a search, visualization,
-dashboard, or index pattern.
-
-To narrow the list of saved objects, enter a search term or select an object 
-type.
+Saved objects store your visualizations, dashboards, maps, Canvas workpads, and more.
```

Comment:
Aren't `saved objects` and `visualizations, dashboards, maps, Canvas workpads, and more` the same thing? 

How about:

`Save your Kibana objects to use at a later time. Objects include visualizations, dashboards, maps, Canvas workpads, and more.`